qiskit/circuit/library/quantum_volume.py

```diff
@@ -12,6 +12,7 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
+# pylint: disable=no-member
```

Comment:
This one doesn't need a disable, since it does not use any monkey-patched method I think. Should we remove it or just generally put it in all files?

Reply:
:+1: Removed.